### PR TITLE
update to eslint-config-loopback@7.0.1

### PR DESCRIPTION
**Pull request by davidcheung**

### CODE REVIEW

File: test/helpers/browser.js [hunk 1]
```diff
@@ -2,7 +2,9 @@
 // Node module: loopback-boot
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
+/*eslint-env es6*/
```
Comment (superkhau) [inline test/helpers/browser.js]: @davidcheung Yes, I agree with eslint env es6 -- @bajtos Already added it specifically to some projects, but we should upgrade eslint-config-loopback like you said.

Comment (bajtos) [inline test/helpers/browser.js]: +1 for upgrading eslint-config-loopback.

For now, you can enable ES6 in `.eslintrc`, see https://github.com/strongloop/loopback-connector-kv-extreme-scale/blob/f2c9ac2813fc446b95b6f4e548826d74f5f93f95/.eslintrc#L3-L5 for an example

Comment (superkhau) [inline test/helpers/browser.js]: https://github.com/strongloop/eslint-config-loopback/pull/14

**Comment by bajtos** [comment]
@davidcheung ping, is this pull request still relevant? Could you please update it to the latest code from `master` and use the latest version of our eslint config?

**Comment by davidcheung** [comment]
@slnode test please

**Comment by bajtos** [comment]
I am closing this pull request as abandoned.

**Pull request closed without merging**